com.aobuchow.themes.spectrum.feature/feature.xml

```diff
@@ -20,10 +20,10 @@
    </license>
 
    <plugin
-         id="com.aobuchow.themes.moderndark"
+         id="com.aobuchow.themes.spectrum"
          download-size="0"
          install-size="0"
-         version="1.0.7.qualifier"
+         version="0.0.0"
```

Comment:
I changed the feature's plugin version to 0.0.0 which is supposed to automatically resolve to the latest?